typescript-test-samples/launchdarkly-lambda-dynamodb/readme.md

```diff
@@ -0,0 +1,90 @@
+
+* Only particular testing or staging environments
+* Only particular users or segments of users
+* Beta testing with managed beta access customers
+* Progressive rollouts to stress testing features
+* Experimentation on features or performance
+
+...and much more. 
+
+## The LaunchDarkly SDKs
+
+Adding LaunchDarkly feature flags to any Lambda always begins with installing the appropriate SDK for the Lambda runtime you are using. You can find a [full list of server-side SDKs here](https://docs.launchdarkly.com/sdk). For the Node.js runtime, that would mean using NPM.
+
+```bash
+npm install launchdarkly-node-server-sdk
+```
+
+If you would like to use [DynamoDB as a persistent feature store](https://docs.launchdarkly.com/sdk/features/storing-data/dynamodb/?q=dynamo), you'll need to also install those dependencies.
+
+```bash
+npm install launchdarkly-node-server-sdk-dynamodb
+```
+
+This works well for a single Lambda, but if you are using feature flags across a number of Lambdas, the easiest method to maintain the depencies across all of them is using a Lambda Layer. The code provided here will assist you in creating the Layer.
+
+### Creating/Updating the Lambda Layer
+
+The necessary dependencies are already included in this project under `/cdk/resources/layers`. It contains both the Node.js SDK and the DynamoDB extension. In order to ensure that this asset has the latest versions of these SDKs, we've included a script that will download the latest versions and archive them into the proper folder. It is recommended that you run this before deploying any of the assets using the provided CDK scripts. All you need to do is:
+
+```bash
+cd  create-layer
+npm run build
+```
+
+## Deploying via the AWS CDK
+
+You'll need to install the CDK. For more details about the AWS CDK and options for installing it, please visit the [AWS docs](https://aws.amazon.com/cdk/). Once the CDK is installed, follow these commands to bootstrap the deployment and then deploy it to your configured AWS account.
+
+```bash
+cd cdk
+cdk bootstrap
+cdk deploy
+```
+
+This will deploy all of the assets discussed above including the Lambda Layer with the LaunchDarkly SDKs and the three example Lambdas, with the Layer as a dependency.
+
+### TypeScript Source Files
+
+Each of the example Lambdas (`index.mjs`) includes the TypeScript source file (`index.ts`) that was used to compile it. These source files include all of the necessary TypeScript types for LaunchDarkly. You can recompile the Lambda source file using the following commands:
+
+```bash
+cd launchdarkly-lambda-dynamodb/cdk
```

Comment:
Lets keep this file path consistent with the rest - the other examples just use `cd cdk` and `cd create-layer`